norsu/extension.py

```diff
@@ -50,6 +65,39 @@ def make(self, targets=None, options=None):
                     cwd=self.work_dir,
                     env=os.environ,
                     output=ExecOutput.Stdout)
+
+            if target in ('check', 'installcheck') and self.specs:
+                print(Style.green('\n$ make isolationcheck'))
+
+                tmpdir = os.path.join(self.work_dir, "tmp_check_iso")
```

Comment:
Maybe we should use `mkdtemp` to make sure that dir name is unique? Otherwise it could result in name clash when used in parallel builds.